### ISSUE TITLE
Doesn't work.

### ISSUE THREAD

**Issue by lodelux** [Ekultek/whitewidow#76]
## Issue/Enhancement/Question (be specific)
Whenever i try to run it with either ruby whitewidow.rb -d or with other arguments, it doesn't work and shows up the following error.

## Exact error message/Enhancement information
Failed with error code (/root/whitewidow/lib/lists/inject.yml): block sequence entries are not allowed in this context at line 12 column 225

## Backtrace from error_log file located in log/log_files/error_log.LOG

["/usr/lib/ruby/2.3.0/psych.rb:379:in `parse'", "/usr/lib/ruby/2.3.0/psych.rb:379:in `parse_stream'", "/usr/lib/ruby/2.3.0/psych.rb:327:in `parse'", "/usr/lib/ruby/2.3.0/psych.rb:254:in `load'", "/usr/lib/ruby/2.3.0/psych.rb:475:in `block in load_file'", "/usr/lib/ruby/2.3.0/psych.rb:474:in `open'", "/usr/lib/ruby/2.3.0/psych.rb:474:in `load_file'", "/root/whitewidow/lib/modules/core/settings.rb:172:in `create_payloads'", "/root/whitewidow/lib/whitewidow/scanner.rb:21:in `get_urls'", "whitewidow.rb:64:in `<main>'"]
["/usr/lib/ruby/2.3.0/psych.rb:379:in `parse'", "/usr/lib/ruby/2.3.0/psych.rb:379:in `parse_stream'", "/usr/lib/ruby/2.3.0/psych.rb:327:in `parse'", "/usr/lib/ruby/2.3.0/psych.rb:254:in `load'", "/usr/lib/ruby/2.3.0/psych.rb:475:in `block in load_file'", "/usr/lib/ruby/2.3.0/psych.rb:474:in `open'", "/usr/lib/ruby/2.3.0/psych.rb:474:in `load_file'", "/root/whitewidow/lib/modules/core/settings.rb:172:in `create_payloads'", "/root/whitewidow/lib/whitewidow/scanner.rb:21:in `get_urls'", "whitewidow.rb:64:in `<main>'"]

## Steps to reproduce if applicable or steps on what should be done
simply run it with -d 

## Search query if applicable (please use exact search query)
inurl:/allindex.php?include=

## User agent (if applicable)
N/A

## Whitewidow version number (must have the actual version run `ruby whitewidow.rb --version`)
2.0.5.5

## Ruby version number (run ruby --version)
2.3.3

## Labels

 - [x] Bug
 - [ ] Encoding
 - [ ] User agent
 - [ ] Enhancement
 - [ ] Question
 - [ ] Email request

**Comment by Ekultek** [comment]
What operating system are you using?

**Comment by lodelux** [comment]
kali linux on virtual machine

**Comment by Ekultek** [comment]
Of course you are lol. Okay give me a couple minutes so I can try to reproduce this

**Comment by Ekultek** [comment]
```
C:\Users\thomas_j_perkins\Documents\bin\ruby\whitewidow>ruby whitewidow.rb -d
 __      __.__    .__  __         __      __.__    .___
/  \    /  \  |__ |__|/  |_  ____/  \    /  \__| __| _/______  _  __
\   \/\/   /  |  \|  \   __\/ __ \   \/\/   /  |/ __ |/  _ \ \/ \/ /
 \        /|   Y  \  ||  | \  ___/\        /|  / /_/ (  <_> )     /
  \__/\  / |___|  /__||__|  \___  >\__/\  / |______ | \____/ \/\_/
       \/       \/              \/      \/           \/    v2.0.5.5(dev)

Clone: https://github.com/WhitewidowScanner/whitewidow.git
Downloads: https://github.com/WhitewidowScanner/whitewidow/releases
[ ! ] Use of this program for malicious intent is illegal. For more information run the --legal flag

[16:46:09 ERROR] Failed with error code (C:/Users/thomas_j_perkins/Documents/bin/ruby/whitewidow/lib/lists/inject.yml): block sequence entries are not allowed in this context at line 12 column 225
[16:46:09 FATAL] Program failed with error code: (C:/Users/thomas_j_perkins/Documents/bin/ruby/whitewidow/lib/lists/inject.yml): block sequence entries are not allowed in this context at line 12 column 225, error saved to error_log.txt
[16:46:09 FATAL] Issue template is being generated for this error, create a new issue named: IYBodFu (C:/Users/thomas_j_perkins/Documents/bin/ruby/whitewidow/lib/lists/inject.yml): block sequence entries are not allowed in this context at line 12 column 225
[16:46:09 INFO] An issue template has been generated for you and is located in C:/Users/thomas_j_perkins/Documents/bin/ruby/whitewidow/tmp/issues/sBud7Ax.txt

C:\Users\thomas_j_perkins\Documents\bin\ruby\whitewidow>
```

Reproduced, I'll look into it and figure out how to fix it. Thanks for brining this to my attention

**Comment by Ekultek** [comment]
fixed via 0f68ed1
